### PR TITLE
Fix compilation warning replace gtk_style_context_get_background_color

### PR DESCRIPTION
**Pull request by zhuyaliang**
Fix compilation warning，Test normal using ```mate-display-properties```

### CODE REVIEW

File: libmate-desktop/mate-bg-crossfade.c [hunk 1]
```diff
@@ -278,7 +294,7 @@ tile_surface (cairo_surface_t *surface,
         gtk_style_context_add_provider (context,
                                         GTK_STYLE_PROVIDER (provider),
                                         GTK_STYLE_PROVIDER_PRIORITY_THEME);
-        gtk_style_context_get_background_color (context, GTK_STATE_FLAG_NORMAL, &bg);
+        get_background_color (context, GTK_STATE_FLAG_NORMAL, &bg);
```
Comment (yetist) [inline libmate-desktop/mate-bg-crossfade.c:297]:
```suggestion
        gtk_style_context_get (context, GTK_STATE_FLAG_NORMAL, "background-color", &bg, NULL);
```

Comment (zhuyaliang) [reply libmate-desktop/mate-bg-crossfade.c]: done

Comment (zhuyaliang) [reply libmate-desktop/mate-bg-crossfade.c]: done

**Review by yetist — APPROVED** [review]
LGTM

**Review by raveit65 — APPROVED** [review]
LGTM,
- warnings are fixed
- changing color desktop backgrounds works fine (mate-background-crossfade)
- changing folder colors in caja works also like before. (mate-desktop-utils)

**Pull request merged**